CONTRIBUTING.md

```diff
@@ -101,19 +101,18 @@ This often provides additional considerations and avoids unnecessary work.
    cd orca
    ```
 
-4. Install `pipx` to easily run Python CLI tools like `tox` and `pipenv`.
-
 4. Create an isolated virtual environment containing package dependencies,
    including those needed for development (*e.g.* testing, documentation) by running:
 
    ```console
-   pipx run tox -e pipenv
+   ./dev_setup.sh <your_python_version>
```

Comment:
Is there a recommended python version to use here for development? Maybe "use the lowest version this project officially supports"?

Reply:
I was planning on just leaving it as use any supported version, which is enforced in the `dev_setup.sh` script. Is there a strong reason to make a specific recommendation?

Also, would you mind holding off on any further review until I mark this as ready for review?

Reply:
> I was planning on just leaving it as use any supported version, which is enforced in the dev_setup.sh script.

Works for me